### PR TITLE
menus.toml: change-shop-link

### PR DESCRIPTION
**Pull request by Stanislaw-bnk**

### CODE REVIEW

**Comment by artur-rs** [comment]
@Stanislaw-bnk at least, `Account`'s url should be changed as well

File: content/markets.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Markets"
+title: "Markets"customer
```
Comment (artur-rs) [inline content/markets.md:2]: @Stanislaw-bnk is it intended?

Comment (Stanislaw-bnk) [reply content/markets.md]: Not exactly, fixed.

Comment (artur-rs) [reply content/markets.md]: fixed here: https://github.com/3mdeb/3mdeb-website/pull/29/commits/848c2b9eeca69d5903aea058577a25f1cc654dab

**Comment by Stanislaw-bnk** [comment]
@artur-rs All set at https://shop.3mdeb.com/shop/

**Pull request merged**